TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/EditInduction/CompletedDate.cshtml.cs

```diff
@@ -1,30 +1,63 @@
+using System.ComponentModel.DataAnnotations;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using TeachingRecordSystem.Core.DataStore.Postgres;
 
 namespace TeachingRecordSystem.SupportUi.Pages.Persons.PersonDetail.EditInduction;
 
 [Journey(JourneyNames.EditInduction), ActivatesJourney, RequireJourneyInstance]
 public class CompletedDateModel : CommonJourneyPage
 {
+    protected TrsDbContext _dbContext;
+    protected IClock _clock;
+
+    public InductionStatus InductionStatus => JourneyInstance!.State.InductionStatus;
+    public string? PersonName { get; set; }
+
+    [BindProperty]
+    [DateInput(ErrorMessagePrefix = "Completed date")]
+    [Required(ErrorMessage = "Enter an induction completed date")]
+    [Display(Name = "When did they complete induction?")]
+    public DateOnly? CompletedDate { get; set; }
+
     public InductionJourneyPage NextPage => InductionJourneyPage.ChangeReasons;
+
     public string BackLink
     {
         // TODO - more logic needed when other routes to completed-date are added
```

Comment:
I think we already have the other routes i.e. coming from the start date page OR from the person induction page OR from the Check Answers page don't we?

Reply:
I haven't added the links from the page - I'll do as another PR if that's OK